README.md

```diff
@@ -10,117 +10,140 @@
 
 ## What is Changelog CI?
 
-Changelog CI is a GitHub Action that generates changelog,
-Then the changelog is prepended to ``CHANGELOG.md`` file and **committed** 
-and/or **commented** to the release Pull request.
+Changelog CI is a GitHub Action that enables a project to utilize an
+automatically generated changelog.
+
+This action will:
+
+* **Prepend** each update to the `CHANGELOG.md` file and
+
+* **Commit** the modified `CHANGELOG.md` file.
+
+* (optionally) **Write** the updates as a comment to the pull request of
+  the corresponding release version.
 
 
 ## How Does It Work:
 
-Changelog CI uses ``python`` and ``GitHub API`` to generate changelog for a repository.
-First, it tries to get the ``latest release`` from the repository (If available).
+Changelog CI uses `python` and `GitHub API` to generate changelog for a repository.
+First, it tries to get the `latest release` from the repository (If available).
 Then, it checks all the pull requests merged after the last release using the GitHub API.
-After that, it parses the data and generates the ``changelog``. Finally,
-It writes the generated changelog at the beginning of the ``CHANGELOG.md`` (or user-provided filename) file.
+After that, it parses the data and generates the `changelog`. Finally,
+It writes the generated changelog at the beginning of the `CHANGELOG.md` (or user-provided filename) file.
 In addition to that, if a user provides a config (JSON file), Changelog CI parses the user-provided config file
 and renders the changelog according to users config.
 Then the changes are **committed** and/or **commented** to the release Pull request.
 
 
 ## Usage:
 
-To use this Action The pull **request title** must match with the default ``regex``
-or the user-provided ``regex`` from the config file.
+To use this Action The pull **request title** must match with the default `regex`
+or the user-provided `regex` from the config file.
 
-**Default Title Regex:** ``^(?i:release)`` (title must start with the word "release" (case insensitive))
+**Default Title Regex:** `^(?i:release)` (title must start with the word "release" (case insensitive))
 
 **Default Version Number Regex:** This Regex will be checked against a Pull Request title.
-This follows [``SemVer``](https://regex101.com/r/Ly7O1x/3/) (Semantic Versioning) pattern.
-e.g. ``1.0.0``, ``1.0``, ``v1.0.1`` etc.
+This follows [`SemVer`](https://regex101.com/r/Ly7O1x/3/) (Semantic Versioning) pattern.
+e.g. `1.0.0`, `1.0`, `v1.0.1` etc.
 
 For more details on **Semantic Versioning pattern** go to this link: https://regex101.com/r/Ly7O1x/3/
 
-**Note:** [you can provide your own ``regex`` through the ``config`` file](#config-file-usage-optional)
+**Note:** You can provide your own `regex` using [the `config` file](#config-file-usage)
```

Comment:
Reference `(#config-file-usage)` does not match with the new header of that section